tests/test_06_l2vpn_return_codes.py

```diff
@@ -497,7 +495,7 @@ def test_054_create_l2vpn_with_all_available_bw(self):
             ],
             "qos_metrics": {
                 "min_bw": {
-                    "value": 89
+                    "value": 5
```

Comment:
I didnt understand this change, Gretel. Can you please elaborate more on this change?